src/functionalTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerControllerTest.java

```diff
@@ -34,5 +40,35 @@ public void given_validEventInformation_should_respond_with_200() {
             .post("/messages")
             .then()
             .statusCode(HttpStatus.NO_CONTENT.value());
+
+
+        taskId = given()
+            .header(SERVICE_AUTHORIZATION, authTokenGenerator.generate())
+            .contentType(APPLICATION_JSON_VALUE)
+            .baseUri(camundaUrl)
+            .basePath("/task")
+            .param("processVariables", "caseReference_eq_" + caseReference)
+            .when()
+            .get()
+            .prettyPeek()
+            .then()
+            .body("size()", is(1))
+            .body("[0].name", is("Process Application"))
+            .body("[0].formKey", is("processApplication"))
+            .extract()
+            .path("[0].id");
+
+    }
+
+    @After
+    public void cleanUpTask() {
+
+        given()
+            .contentType(APPLICATION_JSON_VALUE)
+            .accept(APPLICATION_JSON_VALUE)
+            .header(SERVICE_AUTHORIZATION, authTokenGenerator.generate())
+            .when()
+            .post(camundaUrl + "/task/{task-id}/complete", taskId);
+
```

Comment:
Can we check the state of the completed task? I.e. check that the task no longer exists as part of completing the FT scenario?